### PR TITLE
Dialog display

### PR DESCRIPTION
**Pull request by xoshilcm**

### CODE REVIEW

File: dialog.html [hunk 1]
```diff
@@ -48,6 +70,13 @@ <h2></h2>
         <div id="video-placeholder" class="video"></div>
     </div>
 
+
+    <!-- scripts -->
+    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="dialog.js" type="text/javascript"></script>
+    <script src="https://apis.google.com/js/client.js?onload=init"></script>
```
Comment (scerruti) [inline dialog.html:72]: Do we need to make local copies of these scripts?

**Review by scerruti — COMMENTED** [review]
I'm still confused about this dialog, is this the YouTube playback dialog or the search dialog? It should not be both.

File: dialog.html (hunk 1 above)
Comment (xoshilcm) [reply dialog.html:72]: This is just the search/play YouTube URL dialog. I can change that. I'm not exactly sure if we need local copies of the scripts.

Comment (scerruti) [reply dialog.html:72]: Yes, download all scripts and reference them locally. Some of these may already be in place.

Comment (xoshilcm) [reply dialog.html:72]: I'm not quite sure how to do that. Do I download the scripts and add those files to this branch? Do I need to change anything about the script tags?

**Pull request merged**